Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -52,23 +50,20 @@ public void Constructor_NullArguments_ThrowsArgumentNullException()
         var tfs = Substitute.For<ITfsProcessor>();
         var validator = Substitute.For<ISonarProjectPropertiesValidator>();
 
-        Invoking(() => new PostProcessor(null, null, null, null, null, null)).Should()
+        Invoking(() => new PostProcessor(null, null, null, null, null)).Should()
             .Throw<ArgumentNullException>().And.ParamName.Should().Be("sonarScanner");
 
-        Invoking(() => new PostProcessor(scanner, null, null, null, null, null)).Should()
+        Invoking(() => new PostProcessor(scanner, null, null, null, null)).Should()
             .Throw<ArgumentNullException>().And.ParamName.Should().Be("logger");
 
-        Invoking(() => new PostProcessor(scanner, logger, null, null, null, null)).Should()
+        Invoking(() => new PostProcessor(scanner, logger, null, null, null)).Should()
             .Throw<ArgumentNullException>().And.ParamName.Should().Be("targetUninstaller");
 
-        Invoking(() => new PostProcessor(scanner, logger, targets, null, null, null)).Should()
+        Invoking(() => new PostProcessor(scanner, logger, targets, null, null)).Should()
             .Throw<ArgumentNullException>().And.ParamName.Should().Be("tfsProcessor");
 
-        Invoking(() => new PostProcessor(scanner, logger, targets, tfs, null, null)).Should()
+        Invoking(() => new PostProcessor(scanner, logger, targets, tfs, null)).Should()
             .Throw<ArgumentNullException>().And.ParamName.Should().Be("sonarProjectPropertiesValidator");
-
-        Invoking(() => new PostProcessor(scanner, logger, targets, tfs, validator, null)).Should()
-            .Throw<ArgumentNullException>().And.ParamName.Should().Be("fileWrapper");
```

Comment:
Why was this removed?

Reply:
We don't pass the `fileWrapper` parameter anymore, it was part of the old implementation.